model/CMakeLists.txt

```
@@ -143,6 +143,7 @@ foreach (xlen IN ITEMS 32 64)
                 "riscv_reg_type.sail"
                 "riscv_freg_type.sail"
                 "riscv_regs.sail"
+                "riscv_regs_i.sail"
```

Could add something like 
`option(RVE "Build for RV32E/RV64E as the base isa instead of RV32I/RV64I" OFF)`
```suggestion
                "riscv_regs_$<IF:$<BOOL:RVE>,e,i>.sail"
```
Untested, not sure if the `$<BOOL:` is needed.

---

AFAICT, the `$<BOOL:` is necessary, and the full form is `$<IF:$<BOOL:${RVE}>,e,i>`.  (And people used to give Perl a hard time for being line noise... yeesh.)